questions/questions/_utilities.py

```diff
@@ -26,6 +27,12 @@ def random_answer_options_pitch(len_of_list=4, correct_answer=None):
     random.shuffle(random_answer_list)
     return random_answer_list
 
+def random_root_position_major_triad():
+    pList = [pitch.Pitch('C'), pitch.Pitch('E'), pitch.Pitch('G')]
+    tc = analysis.transposition.TranspositionChecker(pList)
+    transposed_triads = tc.getChordsOfDistinctTranspositions()
+    return random.choice(transposed_triads)
```

Comment:
chord = `[degree.pitch.unicodeName if degree.pitch.accidental.fullName != 'natural' else degree.name for degree in chord]`

A possible solution to eliminating the explicit natural sign when using unicodeName.

Reply:
I'll try this!!!